roles/ovirt-mac-pools/tasks/main.yml

```diff
@@ -0,0 +1,10 @@
+---
+- name: set mac pool
+  ovirt_mac_pools:
+    auth: "{{ ovirt_auth }}"
+    name: "{{ mac_pool_name | default('Default') }}"
```

Comment:
Also if possible please rename `mac_ranges` to `mac_pool_ranges` so we are consistent with mac_pool_name and mac_pool_allow_duplicates